ext/cairo/rb_cairo_font_face.c

```diff
@@ -108,6 +117,11 @@ rb_cairo_font_face_to_ruby_object (cairo_font_face_t *face)
 static VALUE
 cr_font_face_allocate (VALUE klass)
 {
+#if CAIRO_HAS_FT_FONT
+  FT_Error error = FT_Init_FreeType(&library);
```

Comment:
Why would the allocate function be called multiple times per instance? Shouldn't that static instance be different per object instance the same way that `ruby_object_key` is?